src/transaction/esdt.cpp

```diff
@@ -28,4 +57,34 @@ void prepareTransactionForESDTTransfer(Transaction &transaction,
     transaction.m_data = std::make_shared<bytes>(data.begin(), data.end());
 }
 
+void prepareTransactionForESDTIssuance(Transaction &transaction,
+                                      std::string const &token,
+                                      std::string const &ticker,
+                                      std::string const &initialSupply,
+                                      std::string const &noOfDecimals,
+                                      ESDTProperties const &esdtProperties)
+{
+    transaction.m_value = ESDT_ISSUANCE_VALUE;
+    transaction.m_gasLimit = ESDT_ISSUANCE_GAS_LIMIT;
```

Comment:
Perhaps actually compute the gas limit?

Reply:
Yes, that would be way better.
We shall add a functionality for it in the future.